src/ContextMenuTrigger.js

```diff
@@ -51,11 +51,10 @@ export default class ContextMenuTrigger extends Component {
         if (this.props.holdToDisplay >= 0) {
             event.persist();
 
-            this.touchHandled = true;
-
             this.touchstartTimeoutId = setTimeout(
                 () => this.handleContextClick(event),
-                this.props.holdToDisplay
+                this.props.holdToDisplay,
+                this.touchHandled = true
```

Comment:
Woops, that's still not good I'm afraid.. To be fair the short syntax is easy to get confused by, but it should be expanded to this:

```js
             this.touchstartTimeoutId = setTimeout(
                 () => {
                     this.handleContextClick(event);
                     this.touchHandled = true;
                 },
                 this.props.holdToDisplay
             );
```